add markup and styles footer

LiudmylaSK opened the pull request.

LiudmylaSK committed:
add markup and styles footer

mtj-98 committed:
Merge branch 'main' into footer

mtj-98 committed:
Merge branch 'main' into footer

mtj-98 reviewed (changes requested):
Перегляньте будь ласка анімаюцію при ховер і фокусі на контакти, мені чомусь показалось що там не працює)

LiudmylaSK committed:
update markup and styles footer

add id for footer, update stroke-width for icons (tablet/desktop), add transition (var) for contacts and clean code :)

mtj-98 committed:
Merge branch 'main' into footer
